web/controllers/caseload/user_controller.ex

```diff
@@ -22,6 +22,8 @@ defmodule Healthlocker.Caseload.UserController do
   end
 
   def show(conn, %{"id" => id}) do
-    show(conn, %{"id" => id, "section" => "details"})
+    user = Repo.get!(User, id)
```

Comment:
I think that you are right about changing the original link to point to `caseload_user_room_path`. I'm not sure what we need the second `show` action in the `user_controller` in that case though as there will never be a link to it. It will either go to `user_controller` with an `id` and `section` or it will go to the `caseload_controller`.